src/components/Tweet.js

```diff
@@ -1,37 +1,47 @@
-function Tweet() {
+import ProfileImage from "./ProfileImage";
+import User from "./User";
+import Timestamp from "./Timestamp";
+import Message from "./Message";
+import Actions from "./Actions";
+
+
+function Tweet(props) {
+  const {tweet} = props;
   return (
     <div className="tweet">
-      <img
-        src="https://i.imgur.com/9yw1Fyw.jpg"
-        className="profile"
-        alt="profile"
-      />
-
+      <ProfileImage image={tweet.user.image}/>
       <div className="body">
+        <div className="top">
+          <User userData={props.tweet.user} />
+          <Timestamp time={props.tweet.timestamp} />
+        </div>
+        <Message message={props.tweet.message} />
+        <Actions />   
+      </div>
+     {/*  <div className="body">
         <div className="top">
           <span className="user">
-            <span className="name">Ironhack</span>
-            <span className="handle">@ironhack</span>
+            <span className="name">{name}</span>
+            <span className="handle">{handle}</span>
           </span>
 
-          <span className="timestamp">Nov 30, 2020</span>
+          <span className="timestamp">{timestamp}</span>
         </div>
 
         <p className="message">
-          On December 7th, we will be hosting a #webinar that will introduce you
-          to #SQL! Are you ready? 🚀
+          {message}
         </p>
 
         <div className="actions">
-          {/* Font Awesome icons */}
+          
           <i className="far fa-comment"></i>
           <i className="fas fa-retweet"></i>
           <i className="far fa-heart"></i>
           <i className="fas fa-share"></i>
         </div>
       </div>
 
-      <i className="fas fa-ellipsis-h"></i>
+      <i className="fas fa-ellipsis-h"></i> */}
```

Comment:
Uncomment this line so you get the three dots on the top right corner of every tweet